vscode-extension/src/extension.ts

```diff
@@ -128,10 +128,10 @@ export async function activate(context: ExtensionContext) {
   spawn('java', resolveArgs)
     .on('exit', code => {
       if (code !== 0) {
-        console.error(
-          `Could not find Metals server artifact, ensure that metals.serverVersion setting is correct
-           Coursier resolve failed on:${artifact} with exit code:${code}.`
-        );
+        const msg =  `Could not find Metals server artifact, ensure that metals.serverVersion setting is correct.
+                      Coursier resolve failed on:${artifact} with exit code:${code}.`.replace(/^(\s{2})+/gm, '');
```

Comment:
Does this `replace` just removes indentation from the stacktrace? Is it necessary?